rasa/nlu/selectors/response_selector.py

```diff
@@ -414,7 +393,7 @@ def _check_config_params_when_transformer_enabled(self) -> None:
                 f"({self.retrieval_intent})" if self.retrieval_intent else ""
             )
             self._warn_about_transformer_and_hidden_layers_enabled(selector_name)
-            self._warn_and_correct_transformer_size(selector_name)
+            self.component_config[TRANSFORMER_SIZE] = DEFAULT_TRANSFORMER_SIZE
```

Comment:
We don't need this line here as the actual correction happens in `_get_transformer_dimensions`

Reply:
and 4 errors in def test_sets_integer_transformer_size_when_needed, like this:
![Captura de tela de 2022-02-09 23-54-16](https://user-images.githubusercontent.com/52640974/153328360-b13b7da2-4629-4a48-984f-592effadfad7.png)
![Captura de tela de 2022-02-09 23-54-04](https://user-images.githubusercontent.com/52640974/153328363-6722f5e7-e6df-43b3-8280-5e060982a0c1.png)
![Captura de tela de 2022-02-09 23-53-56](https://user-images.githubusercontent.com/52640974/153328365-f81538cb-a2f4-4e01-8c92-1c921501e995.png)
![Captura de tela de 2022-02-09 23-37-26](https://user-images.githubusercontent.com/52640974/153328370-401844d9-95a4-4af1-a9b8-1a4d16a2ae25.png)

Reply:
@jupyterjazz I'm stuck hahaha I'm trying to do this issue with a friend, it was our first issue in Rasa, so we don't know how to fix this tests :(